examples/chip-tool/templates/tests/commands.zapt

```diff
@@ -17,7 +17,7 @@ public:
     TestList() : Command("list") {};
     CHIP_ERROR Run() override
     {
-        {{#chip_tests (getTests)}}printf("{{filename}}\n");{{/chip_tests}}
+        {{#chip_tests "../../../../src/app/tests/suites/ciTests.json"}}printf("{{filename}}\n");{{/chip_tests}}
```

Comment:
As regards to me previous comment, this change should have caused diffs in zzz_generated, but I do not see them.

Reply:
@krypton36 : Why should this cause a change? The whole point is that the new system generates exactly same thing. The same data that was earlier returned by `(getTests)` is now taken from the JSON file.

I've done following tests:
   1.) Remove some tests from ciTests and manualTests, I saw the difference in generated stuff.
   2.) Add some tests back into ciTests  and manualTests, I saw the difference in generated stuff.
   
So it seems to me like what you were concerned about above, is working as it should.

Reply:
As long as the steps above generated differences, was my main concern. Thank you.